src/types/Firestore.ts

```diff
@@ -87,7 +87,7 @@ export interface FirestoreOrdered {
   subvenueEvents?: WithId<VenueEvent>[];
   userModalVisits?: Array<WithId<UserVisit>>;
   userPurchaseHistory?: Array<WithId<Purchase>>;
-  users?: Array<WithId<User>>;
+  universeUsers?: Array<WithId<User>>;
```

Comment:
Name choice for this entity is still in draft

Reply:
As per https://github.com/sparkletown/internal-sparkle-issues/issues/127